From tim.gardner@canonical.com  Fri May  1 14:01:46 2009
From: tim.gardner@canonical.com
Date: Wed, 29 Apr 2009 15:01:42 -0600
Subject: Staging: winbond: mac80211 - unify config_interface and bss_info_changed
To: greg@kroah.com
Cc: Tim Gardner <tim.gardner@canonical.com>
Message-ID: <1241038904-24492-2-git-send-email-tim.gardner@canonical.com>


From: Tim Gardner <tim.gardner@canonical.com>

The commit 'mac80211: unify config_interface and bss_info_changed' from
Johannes Berg <johannes@sipsolutions.net> removed the config_interface structure
tag from struct ieee80211_ops. The BSSID detection functionality migrated to
ieee80211_ops.bss_info_changed.

Since wbsoft_config_interface() was largely empty, there wasn't much to do
other then to remove the function itself. There is currently no support
for BSSID change detection.

Signed-off-by: Tim Gardner <tim.gardner@canonical.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/winbond/wbusb.c |    9 ---------
 1 file changed, 9 deletions(-)

--- a/drivers/staging/winbond/wbusb.c
+++ b/drivers/staging/winbond/wbusb.c
@@ -264,14 +264,6 @@ static int wbsoft_config(struct ieee8021
 	return 0;
 }
 
-static int wbsoft_config_interface(struct ieee80211_hw *dev,
-				   struct ieee80211_vif *vif,
-				   struct ieee80211_if_conf *conf)
-{
-	printk("wbsoft_config_interface called\n");
-	return 0;
-}
-
 static u64 wbsoft_get_tsf(struct ieee80211_hw *dev)
 {
 	printk("wbsoft_get_tsf called\n");
@@ -285,7 +277,6 @@ static const struct ieee80211_ops wbsoft
 	.add_interface		= wbsoft_add_interface,
 	.remove_interface	= wbsoft_remove_interface,
 	.config			= wbsoft_config,
-	.config_interface	= wbsoft_config_interface,
 	.configure_filter	= wbsoft_configure_filter,
 	.get_stats		= wbsoft_get_stats,
 	.get_tx_stats		= wbsoft_get_tx_stats,
